### ISSUE TITLE
S22ultra: HighISO (3200) gets green in raw, jpg looks fine

### ISSUE THREAD

**Issue by nurps** [mirsadm/motioncam#235]
On shots with high ISO the dng files gets very green while the jpg looks ok.

**Comment by mirsadm** [comment]
Is this when you take a photo?

**Comment by nurps** [comment]
Yes photo, sorry, not a video guy. And only in night mode (didn't see that it switched).

Nightmode: ok jpg, green DNG
Photomode: both ok

**Comment by nurps** [comment]
DNG file: https://we.tl/t-vqoOaWMk3U
